samples/openthread/cli/overlay-ant_div.conf

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+# enable radio driver support for Antena Diversity FEM feature
+CONFIG_IEEE802154_NRF5_ANT_DIVERSITY_MODE_AUTO=y
```

Comment:
Why all the NRF5 labels?  Is this a reference to legacy SDK or nRF5x devices?  It seems like an unnecessary qualifier.

Reply:
That's the new api, already available in the ncs. This pr makes use of it for thread samples. We can not change it here:)
My guess is that NRF5 refers to the nRF5 series devices.